src/main/java/no/nav/syfo/exception/ControllerExceptionHandler.java

```diff
@@ -85,7 +85,7 @@ public ControllerExceptionHandler(Metrikk metrikk) {
     private ResponseEntity<ApiError> handleExceptionInternal(Exception ex, ApiError body, HttpHeaders headers, HttpStatus status, WebRequest request) {
         metrikk.tellHttpKall(status.value());
 
-        if (HttpStatus.INTERNAL_SERVER_ERROR.equals(status)) {
+        if (!status.is2xxSuccessful()) {
```

Comment:
Blir ikke dette litt vel mye? Tror i hvert fall ikke vi burde logge 401 som error.

Reply:
Vi kan vel heller logge det som warning 👍 , men feilen burde ikke "slukes" i appen. I dag var det 81 counts på http_status metrikken som skyldtes 400, så kan være greit å litt mer info på hvilken app det feiler mot etc..

Reply:
Ja, sant det. Bare det ikke blir for mye spam så er det jo bra 👍